packages/shared/src/components/squads/Details.tsx

```diff
@@ -191,24 +191,22 @@ export function SquadDetails({
               }}
             />
           )}
-          {createMode && (
-            <div className="flex flex-col justify-start">
-              <h4 className="mb-2 font-bold typo-headline text-theme-label-primary">
-                Post permissions
-              </h4>
-              <p className="mb-4 text-theme-label-tertiary typo-callout">
-                Choose who is allowed to post new content in this Squad.
-              </p>
-              <Radio
-                name="memberPostingRole"
-                options={memberPostingRoleOptions}
-                value={memberPostingRole}
-                onChange={(value) =>
-                  setMemberPostingRole(value as SourceMemberRole)
-                }
-              />
-            </div>
-          )}
+          <div className="flex flex-col justify-start">
```

Comment:
Don't think this adds anything.

```suggestion
          <div className="flex flex-col">
```